R/ModelMeaslesQuarantine.R

```diff
@@ -1,35 +1,44 @@
 #' Measles model with quarantine
 #'
-#' The `ModelMeaslesQuarantine` function implements a connected version
-#' of a Measles model with quarantine.
+#' Implements a Susceptible-Exposed-Infectious-Hospitalized-Recovered (SEIHR)
+#' model for Measles within a school. The model includes isolation of
+#' detected cases and optional quarantine of unvaccinated individuals.
 #'
 #' @param n Number of agents in the model.
 #' @param prevalence Initial number of agents with the virus.
-#' @param contact_rate Average number of contacts per step.
+#' @param contact_rate Average number of contacts per step. Default is set to
+#' match the basic reproductive number (R0) of 15 (see details).
 #' @param transmission_rate Probability of transmission.
 #' @param vax_efficacy Probability of vaccine efficacy.
 #' @param vax_improved_recovery Increase in recovery rate due to vaccination.
 #' @param incubation_period Average number of incubation days.
 #' @param prodromal_period Average number of prodromal days.
 #' @param rash_period Average number of rash days.
-#' @param days_undetected Average number of days undetected.
+#' @param days_undetected Average number of days undetected. Detected cases
+#' are moved to isolation and trigger the quarantine process.
 #' @param hospitalization_rate Probability of hospitalization.
 #' @param hospitalization_period Average number of days in hospital.
 #' @param prop_vaccinated Proportion of the population vaccinated.
 #' @param quarantine_period Total duration of quarantine.
 #' @param quarantine_willingness Probability of accepting quarantine (
 #' see details).
-#' @param isolation_period Average number of days in isolation.
+#' @param isolation_period Total duration of isolation.
```

Comment:
This change would make the line more consistent with the other `*_period` parameter descriptions.
```suggestion
#' @param isolation_period Average number of days in isolation.
```
This is also consistent with what we have done in the `epiworldRShiny` tooltips, however, this reverts a change made in this PR to the `isolation_period` parameter description, so either this stays as "Total duration" and `epiworldRShiny` changes its tooltip, or this changes and `epiworldRShiny` stays the same.